src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+
+.card {
+  display: flexbox;
+  box-sizing: border-box;
+  width: 200px;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__picture {
+    display: block;
+    width: 160px;
+    margin: 0 auto 40px;
+    object-position: center;
+  }
+
+  &__product-title {
+    font-size: $mainElementsSize;
+    font-weight: 500;
+    line-height: $mainLineHeight;
+    margin-block: 0 4px;
```

Comment:
you can just `margin-bottom: 4px;` here and on other places below

Reply:
then it adds extra space on top and I don't know why